src/sentry/search/django/backend.py

```diff
@@ -76,8 +75,9 @@ class ScalarCondition(Condition):
     instances
     """
 
-    def __init__(self, field):
+    def __init__(self, field, extra_q=None):
```

Comment:
I'd probably call this something other than `extra_q`, since this isn't a `Q` object